app/views/application/_footer.html.slim

```diff
@@ -1,5 +1,16 @@
-footer
-  <iframe src="https://ghbtns.com/github-btn.html?user=codetriage&repo=codetriage&type=watch&count=true&v=2" frameborder="0" scrolling="0" width="100px" height="20px"></iframe>
-  <iframe src="https://ghbtns.com/github-btn.html?user=codetriage&repo=codetriage&type=star&count=true" frameborder="0" scrolling="0" width="100px" height="20px"></iframe>
-  <iframe src="https://ghbtns.com/github-btn.html?user=codetriage&repo=codetriage&type=fork&count=true" frameborder="0" scrolling="0" width="100px" height="20px"></iframe>
-  p.text-center &copy CodeTriage 2015
+footer.application-footer
+  hr.section-break
+
+  = link_to "Submit a Repo", new_repo_path, class: 'button footer-action'
+
+  .github-numbers
+    <iframe src="https://ghbtns.com/github-btn.html?user=codetriage&repo=codetriage&type=watch&count=true&v=2" frameborder="0" scrolling="0" width="100px" height="20px"></iframe>
+    <iframe src="https://ghbtns.com/github-btn.html?user=codetriage&repo=codetriage&type=star&count=true" frameborder="0" scrolling="0" width="100px" height="20px"></iframe>
+    <iframe src="https://ghbtns.com/github-btn.html?user=codetriage&repo=codetriage&type=fork&count=true" frameborder="0" scrolling="0" width="100px" height="20px"></iframe>
```

Comment:
I'd be tempted to create a helper for these, just because I had to look so hard to see where the difference was.